### PR TITLE
Cheque print

### PR DESCRIPTION
**Pull request by saurabh6790**
## Cheque Printing 
- Settings 

screen shot 2016-05-16 at 12 32 56 pm
- Print Format

screen shot 2016-05-16 at 12 33 13 pm
- Preview
  
  Preview on settings page

screen shot 2016-05-16 at 12 34 49 pm
 <br>

```
  Print Preview
```

screen shot 2016-05-16 at 12 33 40 pm

### CODE REVIEW

File: erpnext/accounts/doctype/cheque_print_template/cheque_print_template.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		})
+	else:
+		cheque_print = frappe.get_doc("Print Format", template_name)
+	
+	doc = frappe.get_doc("Cheque Print Template", template_name)
+	
+	cheque_print.html = """
+<div style="position: relative; top:%scm">
+<div style="width:%scm;height:%scm;">
+	<span style="top:%s cm; left:%scm;position: absolute;">
```
Comment (rmehta) [inline erpnext/accounts/doctype/cheque_print_template/cheque_print_template.py]: better to use dict here `%(starting_position_from_top_edge)s` for debugging

File: erpnext/accounts/doctype/cheque_print_template/cheque_print_template.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	refresh: function(frm) {
+		if(!frm.doc.__islocal) {
+			frm.add_custom_button(frm.doc.has_print_format?__("Update Print Format"):__("Create Print Format"),
+				function() {
+					erpnext.cheque_print.view_cheque_print(frm);
+				}).addClass("btn-primary");
+				
+			$(frm.fields_dict.cheque_print_preview.wrapper).empty()
+			
+			$("<div style='position: relative; overflow-x: scroll;'>\
```
Comment (rmehta) [inline erpnext/accounts/doctype/cheque_print_template/cheque_print_template.js]: Use js templating

**Comment by saurabh6790** [comment]
@rmehta fixed

**Comment by nabinhait** [comment]
Payer or Payee?

**Pull request merged**